boards/arm/actinius_icarus/actinius_icarus_common.dts

```diff
@@ -47,7 +66,9 @@
 		led0 = &red_led;
 		led1 = &green_led;
 		led2 = &blue_led;
-		rgb-pwm = &pwm0;
+		red-pwm-led = &red_pwm_led;
```

Comment:
Why not add also at least `pwm-led0` alias here?

Reply:
@anangl Added. Thanks.